src/context.ts

```diff
@@ -20,21 +21,25 @@ export default class Context {
    * Build context by scanning directories
    * @param paths - directory with classes
    */
-  public async build(...paths: string[]): Promise<void> {
-    for (const dir of paths) {
-      const filenames: string[] = await Utils.listDir(path.resolve(__dirname, dir));
-      for (const absFilename of filenames) {
-        if (absFilename.endsWith('.map')) {
-          continue;
-        }
-        try {
-          // get default exported class
-          // eslint-disable-next-line @typescript-eslint/no-var-requires
-          const clazz = require(absFilename).default;
-          this.register(new clazz());
-        } catch (e) {
-          // do nothing
+  public async build(...paths: (Contextual | string)[]): Promise<void> {
+    for (const elem of paths) {
+      if (typeof elem === 'string') {
+        const filenames: string[] = await Utils.listDir(path.resolve(__dirname, elem));
+        for (const absFilename of filenames) {
+          if (absFilename.endsWith('.map') || absFilename.endsWith('.test.ts')) {
```

Comment:
If I understand correctly this logic attempts to load files from the filesystem and instantiate classes based on them?
This seems pretty unsafe, especially using this kind of logic to filter files out, for example just adding a `.gitignore` file in one of these folders would break it, no?

Reply:
I guess it will not fail. Pls look at the lines beneath:

```javascript
         try {
            // get default exported class
            // eslint-disable-next-line @typescript-eslint/no-var-requires
            const clazz = require(absFilename).default;
            this.register(new clazz());
          } catch (e) {
            // do nothing
          }
```